docs/index.md

```diff
@@ -0,0 +1,138 @@
+Paperless, check out these resources in the documentation:
+
+- `[Some screenshots](#screenshots) of the new UI are available.
+- Read [this section](/advanced_usage/#advanced-automatic_matching) if you want to learn about how paperless automates all
+  tagging using machine learning.
+- Paperless now comes with a `[proper email consumer](/usage/#usage-email) that's fully tested and production ready.
+- Paperless creates searchable PDF/A documents from whatever you put into the consumption directory. This means
+  that you can select text in image-only documents coming from your scanner.
+- See `[this note](/administration/#utilities-encyption) about GnuPG encryption in paperless-ngx.
+- Paperless is now integrated with a
+  [task processing queue](/setup#task_processor) that tells you at a glance when and why something is not working.
+- The [changelog](/changelog) contains a detailed list of all changes in paperless-ngx.
+
+## Screenshots
+
+This is what Paperless-ngx looks like.
+
+The dashboard shows customizable views on your document and allows
+document uploads:
+
+[![image](assets/screenshots/dashboard.png)](assets/screenshots/dashboard.png)
+
+The document list provides three different styles to scroll through your
+documents:
+
+[![image](assets/screenshots/documents-table.png)](assets/screenshots/documents-table.png)
+
+[![image](assets/screenshots/documents-smallcards.png)](assets/screenshots/documents-smallcards.png)
+
+[![image](assets/screenshots/documents-largecards.png)](assets/screenshots/documents-largecards.png)
+
+Paperless-ngx also supports dark mode:
+
+[![image](assets/screenshots/documents-smallcards-dark.png)](assets/screenshots/documents-smallcards-dark.png)
+
+Extensive filtering mechanisms:
+
+[![image](assets/screenshots/documents-filter.png)](assets/screenshots/documents-filter.png)
+
+Bulk editing of document tags, correspondents, etc.:
+
+[![image](assets/screenshots/bulk-edit.png)](assets/screenshots/bulk-edit.png)
+
+Side-by-side editing of documents:
+
+[![image](assets/screenshots/editing.png)](assets/screenshots/editing.png)
+
+Tag editing. This looks about the same for correspondents and document
+types.
+
+[![image](assets/screenshots/new-tag.png)](assets/screenshots/new-tag.png)
+
+Searching provides auto complete and highlights the results.
+
+[![image](assets/screenshots/search-preview.png)](assets/screenshots/search-preview.png)
+
+[![image](assets/screenshots/search-results.png)](assets/screenshots/search-results.png)
+
+Fancy mail filters!
+
+[![image](assets/screenshots/mail-rules-edited.png)](assets/screenshots/mail-rules-edited.png)
+
+Mobile devices are supported.
+
+[![image](assets/screenshots/mobile.png)](assets/screenshots/mobile.png)
+
+## Support
+
+Community support is available via [GitHub Discussions](https://github.com/paperless-ngx/paperless-ngx/discussions/categories/feature-requests) and [the Matrix chat room](https://matrix.to/#/#paperless:matrix.org).
```

Comment:
Maybe https://github.com/paperless-ngx/paperless-ngx/discussions is a more general link in this case?